backend/Tournament/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.utils import timezone
+
+from User.models import User
+from .tournamentState import TournamentState
+
+# Create your models here.
+class Tournament(models.Model):
+	host_user = models.ForeignKey(User, related_name='tournament_hosts', on_delete=models.CASCADE)
```

Comment:
Does this mean with the cascade that we run into problems deleting the user to then dissolve the tournaments as well?